templates/includes/sidebar.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block sidebar %}
+  <ul>
+    <li>
+      <a href="#"{% url "taxi:index" %}>Home page</a>
```

Comment:
Please review the task checklist again: 

Write realization of sidebar include that must have a list of empty links:
    * Home page
    * Manufacturers
    * Cars
    * Drivers